src/Symfony/Bridge/Twig/NodeVisitor/Scope.php

```diff
@@ -52,7 +52,7 @@ public function enter()
     /**
      * Closes current scope and returns parent one.
      *
-     * @return Scope|null
+     * @return self|null
```

Comment:
>Personally, i prefer it...

I don't 😆 

I just don't understand the motivations behind, and there are many places where this was not changed.